### DISCUSSION TITLE
doris跑tpch并发设置8、16、32、64、96，结果都是10分钟左右

### DISCUSSION THREAD

**Discussion by dingjing0604** [apache/doris#8130] [A - General / Q&A]
配置 1FE+3BE，高效ESSD盘。测试场景为TPCH100G。
设置并发度为1时，需要22分钟。设置并发度为8、32、64、96时，时间基本都是10分钟左右。
请问这是正常的吗？
为什么别的数据库跑TPCH的时候会在某一并发度取得最佳性能，但是doris的结果都一致？
（以上测试结果都是在同一并发度下跑10次取得的最优结果）

**Comment by BiteTheDDDDt** [comment]
doris并发是fragment级别的 如果查询没有规划出那么多fragment的话并发度开大也没有用

**Reply by zyw8136** [reply]
@BiteTheDDDDt 您好，不太明fragment级别是什么意思，能给解释一下吗？

**Reply by BiteTheDDDDt** [reply]
你可以在你的查询前面加explain graph 就能看到规划出来的fragment tree的形态了
